src/uriHandler.test.ts

```diff
@@ -32,4 +32,46 @@ describe("UriEventHandler - /projectScaffold", () => {
     );
     assert.strictEqual(projectScaffoldUriStub.firstCall.args[0].toString(), uri.toString());
   });
+
+  it("should throw an error when collection or template not present", async () => {
+    const uriMissingTemplate = vscode.Uri.parse(
+      "vscode://confluentinc.vscode-confluent/projectScaffold?collection=myCollection&cc_bootstrap_server=myBootstrapServer&cc_api_key=myApiKey&cc_api_secret=myApiSecret&cc_topic=myTopic",
+    );
+
+    const consoleErrorStub = sandbox.stub(console, "error");
```

Comment:
No need to stub this if we aren't checking for it in the test(s), and we should probably use the `Logger` class to make sure any error messages show up in the main `Confluent` output channel.